sdk/ml/azure-ai-ml/azure/ai/ml/operations/_batch_deployment_operations.py

```diff
@@ -39,7 +39,7 @@
 
 ops_logger = OpsLogger(__name__)
 logger, module_logger = ops_logger.package_logger, ops_logger.module_logger
-
+DeploymentType = TypeVar("DeploymentType", bound=Union[BatchDeployment, PipelineComponentBatchDeployment])
```

Comment:
Do we need to update this to include `ModelBatchDeployment`?

Reply:
No, given that ModelBatchDeployment and BatchDeployment are follow the same creation flow